### PR TITLE
Remove attributes in `_sdl2.video`, turn them into setters/getters

### PR DESCRIPTION
**Pull request by yunline**

### CODE REVIEW

**Comment by ankith26** [comment]
Thanks for the PR!

Not sure whether this is the right way to proceed, sure it makes the API more consistent with the older `Surface` API, but personally I think attributes are a more "modern" way of doing things, and any future API we add should follow the newer trends.

Ideally this sort of PRs generally need an issue discussing this first with other fellow contributors (or was it done already elsewhere and I missed it? Please do correct me if I missed it)

**Comment by yunline** [comment]
> Thanks for the PR!
> 
> Not sure whether this is the right way to proceed, sure it makes the API more consistent with the older `Surface` API, but personally I think attributes are a more "modern" way of doing things, and any future API we add should follow the newer trends.
> 
> Ideally this sort of PRs generally need an issue discussing this first with other fellow contributors (or was it done already elsewhere and I missed it? Please do correct me if I missed it)

Issue: https://github.com/pygame-community/pygame-ce/issues/1932
Discord: https://discord.com/channels/772505616680878080/772940667231928360/1079359592926040146

**Comment by yunline** [comment]
> Thanks for the PR!
> 
> Not sure whether this is the right way to proceed, sure it makes the API more consistent with the older `Surface` API, but personally I think attributes are a more "modern" way of doing things, and any future API we add should follow the newer trends.
> 
> Ideally this sort of PRs generally need an issue discussing this first with other fellow contributors (or was it done already elsewhere and I missed it? Please do correct me if I missed it)

Issue: https://github.com/pygame-community/pygame-ce/issues/1932
Discord: https://discord.com/channels/772505616680878080/772940667231928360/1079359592926040146

File: buildconfig/stubs/pygame/_sdl2/video.pyi [hunk 1]
```diff
@@ -51,27 +51,35 @@ class Window:
     ) -> None: ...
     @classmethod
     def from_display_module(cls) -> Window: ...
-    grab: bool
-    relative_mouse: bool
+    def get_geab(self) -> bool: ...
```
Comment (Mega-JC) [inline buildconfig/stubs/pygame/_sdl2/video.pyi:54]:
```suggestion
    def get_grab(self) -> bool: ...
```

**Comment by Mega-JC** [comment]
This PR covers converting `@property` attributes into separate getters and setters, but ignores other attributes. This leads to some odd API differences. Ideally, `Image.alpha` should be `Image.get_alpha()` and `Image.set_alpha()`, the same also goes for `Image.blend_mode`.

**Pull request closed without merging**